packages/webawesome/docs/assets/styles/utils.css

```
@@ -99,6 +99,15 @@
     line-height: var(--wa-form-control-value-line-height);
     padding-inline: 0.33em;
   }
+
+  /* step icons for ordered instructions */
+  .step-icon {
+    --primary-color: var(--wa-color-neutral-on-normal);
+    --secondary-color: var(--wa-color-neutral-fill-normal);
+    --secondary-opacity: 1;
+    font-size: var(--wa-font-size-2xl);
```

```suggestion
    font-size: var(--font-size, var(--wa-font-size-2xl));
```

NIT: Can we set a default font-size here using a custom prop so that it can be overriden depending on context? I could see us using this icon in various heading levels (with it scaling to match).

---

Thinking about the idea of this icon scaling based on context, should this rule set a `font-size`?

If we wanted to override the `inherit` behavior with a specific size, that seems to be a situational call best handled by custom or inline styling, no?

---

Perhaps the best of both worlds: I updated `.step-icon` to use a relative size (`font-size: 2em;`). This way, they're larger than the text by default (which so far has been the preference in 2FA and here in the Figma doc), but it adapts to the surrounding text and we can still override as the situation demands. Does that sit right with you?